app/models/public_body.rb

```diff
@@ -151,12 +152,11 @@ def translated_versions
         translations
     end
 
-    def translated_versions=(translation_attrs)
+    def translations_attributes=(translation_attrs)
         def empty_translation?(attrs)
-            attrs_with_values = attrs.select{ |key, value| value != '' and key != 'locale' }
+            attrs_with_values = attrs.select{ |key, value| value != '' and key.to_s != 'locale' }
```

Comment:
I have a feeling there was an issue with this. Just making a note to check IRC history.